src/rules/selector-max-specificity/index.js

```diff
@@ -0,0 +1,46 @@
+// Bring in dependencies
+import { isNumber } from "lodash"
+import { calculate }  from "specificity"
+import {
+  report,
+  ruleMessages,
+  validateOptions,
+} from "../../utils"
+
+export const ruleName = "selector-max-specificity"
+
+export const messages = ruleMessages(ruleName, {
+  expected: (selector, specificity) => `Expected "${selector}" to have a specificity equal to or less than "${specificity}"`,
+})
+
+export default function (max) {
+  return (root, result) => {
+    const validOptions = validateOptions(result, ruleName, {
+      actual: max,
+      possible: [isNumber],
```

Comment:
Can be more specific with this, right? It has a max of four digits, cannot have decimals, etc.